recipes/jupyter-server-documents/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jupyter_server_documents" %}
+{% set version = "0.1.0a5" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
It is not about building fine or not. It is about stability.  Development releases should not be in the stable channel. You can follow CFEP-05 or ask upstream for a stable release. Note that, PyPI has a similar mechanism. People must pin to the dev release to get it, otherwise they'll get the stable one available. IMO their implementation is a bit confusing, but works too. Conda-forge's is more explicit and not Python exclusive.

Reply:
Thank you! I've applied the updates.